tests/testthat/test-queue.R

```diff
@@ -32,7 +32,7 @@ test_that("queue works as intended", {
   expect_equal(status$queue, 0)
 
   ## After task has completed
-  Sys.sleep(1)
+  Sys.sleep(2)
```

Comment:
You can use task_wait on the underlying rrq object here if you want to wait for less time and make the test less fragile, and we could reduce the sleep time further as that's not really needed here?